Softeq.XToolkit.Common/Softeq.XToolkit.Common.csproj

```diff
@@ -1,9 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>netstandard2.0</TargetFramework>
+		<TargetFramework>netstandard2.1</TargetFramework>
 		<AssemblyName>Softeq.XToolkit.Common</AssemblyName>
 		<RootNamespace>Softeq.XToolkit.Common</RootNamespace>
-		<LangVersion>7.3</LangVersion>
+	</PropertyGroup>
+	<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+	  <LangVersion>8.0</LangVersion>
+	  <Nullable>enable</Nullable>
+	</PropertyGroup>
```

Comment:
Please set lang version and `<Nullable>` in the shared `<PropertyGroup>`, instead of build configurations (VS default behavior)